service/controller/key/cluster.go

```diff
@@ -86,6 +87,16 @@ const (
 	ComponentOS = "containerlinux"
 )
 
+const (
+	// Defines how many times node-auto-repair logic needs to see node
+	// in NotReady state before it will terminate it to repair it.
+	NodeNotReadyTickThreshold = 4
```

Comment:
Would be nice to have an explanation of a "tick" here. Is it one reconciliation loop?